collect_app/src/main/java/org/odk/collect/android/utilities/ResponseMessageParser.java

```diff
@@ -0,0 +1,87 @@
+package org.odk.collect.android.utilities;
+
+import org.opendatakit.httpclientandroidlib.HttpEntity;
+import org.opendatakit.httpclientandroidlib.util.EntityUtils;
+import org.w3c.dom.Document;
+import org.xml.sax.SAXException;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+
+/**
+ * Created by Jon Nordling on 2/21/17.
+ * The purpose of this class is to handle the XML parsing
+ * of the server responses
+ */
+
+public class ResponseMessageParser {
+    private static HttpEntity httpEntity;
+    private static final String MESSAGE_XML_TAG = "message";
+    public static Boolean isValid = false;
+
+    public static String messageResponse;
+
```

Comment:
All of the fields should be non-static and private since you're using them as instance variables and have getters for them.